components/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -46,7 +46,7 @@ export const ThemeSwitcher = () => {
 		document.addEventListener("keyup", handleThemeSwitcherKB)
 
 		return () => document.removeEventListener("keyup", handleThemeSwitcherKB)
-	}, [])
+	})
```

Comment:
The array is there to make sure the event listeners are only rendered when the component mounts. By removing the array the useEffect runs every time anything rerenders which is not the behaviour wanted. Either an eslint comment should be used to ignore the empty array or the function can be added to the array and wrapped in a useCallback function so it doesn't get recreated on every render.

Reply:
I have to read more for this one, will let ya know soon

Reply:
took the eslint disable way